src/BookReader.js

```diff
@@ -662,87 +671,140 @@ BookReader.prototype.resize = function() {
 };
 
 /**
- * Binds keyboard event listeners
+ * Returns true if passed element activates or maintains BookReader keyboard shortcut focus
```

Comment:
Does this mean when users interact with these elements like the `item-navigator`, `#topnav` and `#donate_banner`, the BookReader keyboard shortcuts continue to work? (and conversely do other elements cause it them to stop?)

Reply:
Yes

Reply:
Gotcha. So when users interact with those elements, do they have the ability to use keyboard navigation on them?

Reply:
Yes, 
- different keys -- or so I thought -- retesting, the donation banner radio buttons use up/down/right/left keys. These need to _not_ be take over as keyboard shortcuts.